### ISSUE TITLE
Unable to Edit/Delete template region

### ISSUE THREAD

**Issue by akash0234** [cofoundry-cms/cofoundry#549]
![image](https://github.com/user-attachments/assets/d5793931-2c36-4029-909d-a64664a60b6a)

There is no icon cming to modify the region block. i have
await Cofoundry.Template.CustomEntityRegion("Body").AllowMultipleBlocks().EmptyContentMinHeight(500).InvokeAsync()

set in my page template

**Comment by HeyJoel** [comment]
Your block code references custom entity regions, but the screen shot looks like you're ediing a regular page. Did you mean to use `Cofoundry.Template.Region`?:

```
@(await Cofoundry.Template.Region("Body")
    .AllowMultipleBlocks()
    .EmptyContentMinHeight(500)
    .InvokeAsync())
```

See [Page Templates](https://www.cofoundry.org/docs/content-management/page-templates) vs [Custom Entity Pages](https://www.cofoundry.org/docs/content-management/custom-entity-pages).

Custom entity regions will only be editable on a custom entity page. You can also add regular regions to custom entity pages, in which case you'll get two edit buttons in the UI, one to edit the page template and one to edit the custom entity page.

**Comment by akash0234** [comment]
Its a Custom Entity Page.. Blog Details ,,  In the details page I have  Template.Region("Body") as well as CustomEntityRegion Still I cant see any buttons

**Comment by HeyJoel** [comment]
The [Simple Site example](https://github.com/cofoundry-cms/Cofoundry.Samples.SimpleSite) works just fine for me and contains both custom entity pages and regular page templates, so perhaps you can see if this works for you and use it as a reference.

You're not providing enough information here to resolve the issue. I suggest that you try and create a [minimal reproducable example](https://stackoverflow.com/help/minimal-reproducible-example), or just strip back your page to the basic html only and gradually introduce code to see where the issue is e.g. regions, css, javascript.

Some other things to think about:

- What version of Cofoundry/.NET are you using?
- Did it ever work? Have you got pages elsewhere with regions that work?
- If you go to the templates page in the admin panel, is it showing that your regions are registered?
- Are there any related JS or resource network errors in your browser dev tools?
- What browser are you using?

**Comment by akash0234** [comment]
Coundry .Net 6.

It used to work properly.
I can see regions in the Blog details Admin.
I dont see any errors in the
I'm using chrome

**Comment by akash0234** [comment]
I don't know how but

Somehow calling async functions inside custom Entity details page causing This button disappearance issue,

![image](https://github.com/user-attachments/assets/90c5127e-ece6-49dc-9c8e-6c3ac1b16038)

Finaly When i used normal ajax with no async code It started working like before..

![image](https://github.com/user-attachments/assets/72c4eb00-8993-4235-adc0-75d303bcbe4a)

I'm Closing the thread.. Thanks for your quick replies. ur awesome as always

**Comment by HeyJoel** [comment]
I think the problem is that you are assigning window.onload rather than binding with addEventListener. Unfortunately it looks like we're doing the same internally so you're overwriting our event binding. We should fix that in our code but I'll have a proper look tomorrow and verify.

**Comment by HeyJoel** [comment]
Ok, yes that was the problem and it's now fixed in v0.12.1. Thanks for responding with your solution, it triggered the fix.
